level1/ddhi.odd

```diff
@@ -30,13 +30,13 @@
 
 	  <p>
 	    The first section is a general-purpose description of the
-	    DDHI schema and how projects should use it.  It is
+	    DDHI schema and how projects and researchers should use it.  It is
```

Comment:
By "projects" I mean to include researchers who are encoding their own texts.  These guidelines are not aimed at people who want to use the corpus.